### PR TITLE
SHIP-0001 Previous Proposals

### PR DESCRIPTION
**Pull request by adambkaplan**
Document the previous proposals merged in the shipwright-io/build
repository.

### CODE REVIEW

File: ships/0001-previous-proposals.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+## Summary
+
+This is a summary of the merged proposals that pre-dated the SHIP process.
+Full details of these proposals can be found in [shipwright-io/build/docs/proposals](https://github.com/shipwright-io/build/tree/main/docs/proposals).
+
+## Motivation
+
+As of May 24, 2021, the community repository will serve as the official record of all new Shipwright enhancement proposals.
+Previously discussed and merged proposals will not be migrated here - they will remain in shipwright-io/build for posterity.
```
Comment (otaviof) [inline ships/0001-previous-proposals.md:44]: What's the motivation to keep them on the controller repository? Wouldn't it be more practical to copy them over here (this repo) and keep them all together?

Comment (otaviof) [reply ships/0001-previous-proposals.md:44]: (And after copying, removing them from the Build-Controller repo).

**Review by qu1queee — APPROVED** [review]
/lgtm

File: ships/0001-previous-proposals.md (hunk 1 above)
Comment (adambkaplan) [reply ships/0001-previous-proposals.md:44]: Moving everything over presented its own challenges, particularly for proposals currently under discussion. Felt easier to keep the old in the controller repo, and have new proposals after 5/24/2021 live here.

**Comment by adambkaplan** [comment]
/approve

in light of an LGTM from an approver.

**Pull request merged**